src/gui/gui_console.cpp

```diff
@@ -313,7 +317,7 @@ parser_state parse_command(sys::state& state, std::string_view text) {
 	parser_state pstate{};
 	pstate.cmd = possible_commands[0];
 	for(auto const& cmd : possible_commands)
-		if(s.starts_with(cmd.name)) {
+		if(s.compare(0, cmd.name.size(), cmd.name) == 0) {
```

Comment:
won't this have the same problem? It will compare the first cmd.name.size() characters of cmd to s. In other words, it will return true if cmd is a prefix of s, which is what starts with was doing, no?

Reply:
yes

Reply:
already realized